samples/bluetooth/peripheral_bms/README.rst

```diff
@@ -0,0 +1,102 @@
+.. _peripheral_bms:
+
+Bluetooth: Peripheral Bond Management Service (BMS)
+###################################################
+
+The peripheral BMS sample demonstrates how to use the :ref:`bms_readme`.
+
+Overview
+********
+
+When connected, the sample waits for Client requests to perform any bond deleting operation.
+
+It supports up to two simultaneous Client connections.
+
+Requirements
+************
+
+* One of the Nordic development boards:
```

Comment:
Include the supported DKs from sample_boardnames.txt

Reply:
fixed